scripts/promote-develop-to-staging.sh

```diff
@@ -34,3 +34,4 @@ echo "Pushing develop branch..."
 git push --force # force required in case staging had new content commits (expected)
 
 echo "Promotion Complete!"
+ 
```

Comment:
This newline at the end of file should be maintained https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.

Alternatively just drop this change from the changeset.

Reply:
thanks for that. Not sure how that happened, I had been inspecting the file for something unrelated and it got caught up on that commit.